### DISCUSSION TITLE
draw breaks with ComplexHeatmap::pheatmap

### DISCUSSION THREAD

**Discussion by flde** [jokergoo/ComplexHeatmap#804] [Q&A]
Hi, 

if I try to move annotations with draw() it does not work when using ComplexHeatmap::pheatmap. For reproducibility I used the code from #776 and just changed Heatmap() to ComplexHeatmap::pheatmap. I could not find any info about that. 

Should I provide more specific code or do you have already a idea what is going on?

Best, 
Flo

**Comment by jokergoo** [comment]
When you mentioned "changed Heatmap() to ComplexHeatmap::pheatmap", how did you do that? The two functions (`Heatmap()` and `ComplexHeatmap::pheatmap()` have different sets of arguments. 

If would be good if you can provide example code.

**Reply by flde** [reply]
Sorry for my late reply I was on parental leave. Many thanks for your help @jokergoo! I continued without ordering the labels but will come back if I have the code to compare the two functions.
